From 7e102b1eb2ca3eff7a6f33ebeab17825e6f70956 Mon Sep 17 00:00:00 2001
From: Robert Marko <robimarko@gmail.com>
Date: Mon, 4 Nov 2024 22:01:24 +0100
Subject: [PATCH] arm64: dts: qcom: ipq6018: add NSS reserved memory

It seems that despite NSS not being supported in OpenWrt the memory it
usually uses needs to be reserved anyway for stability reasons.

Signed-off-by: Robert Marko <robimarko@gmail.com>
---
 arch/arm64/boot/dts/qcom/ipq6018.dtsi | 5 +++++
 1 file changed, 5 insertions(+)

--- a/arch/arm64/boot/dts/qcom/ipq6018.dtsi
+++ b/arch/arm64/boot/dts/qcom/ipq6018.dtsi
@@ -194,6 +194,11 @@
 			no-map;
 		};
 
+		nss_region: memory@40000000 {
+			reg = <0x0 0x40000000 0x0 0x01000000>;
+			no-map;
+		};
+
 		bootloader@4a100000 {
 			reg = <0x0 0x4a100000 0x0 0x400000>;
 			no-map;
@@ -218,6 +223,16 @@
 			reg = <0x0 0x4ab00000 0x0 0x5500000>;
 			no-map;
 		};
+
+		q6_etr_region: q6_etr_dump@50000000 {
+			reg = <0x0 0x50000000 0x0 0x100000>;
+			no-map;
+		};
+
+		m3_dump_region: m3_dump@50100000 {
+			reg = <0x0 0x50100000 0x0 0x100000>;
+			no-map;
+		};
 	};
 
 	smem {
